### PR TITLE
Create CODE_OF_CONDUCT

### PR DESCRIPTION
**Pull request by Mudrank**

### CODE REVIEW

File: docs/CODE_OF_CONDUCT [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+an individual is representing the project or its community in public spaces.
+Examples of representing a project or community include using an official
+project e-mail address, posting via an official social media account, or acting
+as an appointed representative at an online or offline event. Representation of
+a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting [heyBereket](https://twitter.com/heybereket) or [Landon](https://twitter.com/theldb_)  on Twitter ,
```
Comment (heybereket) [inline docs/CODE_OF_CONDUCT:59]: Could you remove Landon from here? I'm the original creator, I don't think he'd want to deal with this.

**Review by heybereket — CHANGES_REQUESTED** [review]
Amazing, thank you! Left 1 piece of feedback:

**Comment by heybereket** [comment]
This seems like a copy from Ben Awad's Dogehouse, I'm going to have to decline.

**Comment by Mudrank** [comment]
>
 This seems like a copy from Ben Awad's Dogehouse, I'm going to have to decline.

Hey we use the template given by contributor-covenant.org you can check it [here](https://www.contributor-covenant.org/version/1/4/code-of-conduct.html)

**Pull request closed without merging**